app/main/appium-method-handler.js

```diff
@@ -3,6 +3,7 @@ import wd from 'wd';
 import log from 'electron-log';
 import _ from 'lodash';
 import { SCREENSHOT_INTERACTION_MODE } from '../renderer/components/Inspector/shared';
+import {getWebviewStatusAddressBarHeight, setHtmlElementAttributes} from './helpers';
```

Comment:
This import may break npm run test. I think some electron dependencies haven't been ready in the test code, but here tried to import them.

Reply:
Probably a new file for getWebviewStatusAddressBarHeight and setHtmlElementAttributes and import them may fix this import error

Reply:
Fixed it, thanks